documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1254,26 +1270,22 @@ In the case of the Oracle connector, the structure includes the following fields
 * The {prodname} version.
 * The connector name.
 * Whether the event is part of an ongoing snapshot or not.
-* The transaction id (not includes for snapshots).
-* The SCN of the change.
+* The transaction id (not included for snapshots).
+* The `commit_scn` for the change.
```

Comment:
```suggestion
* The `scn` for the change and the `commit_scn` for enclosing transaction.
```
We should probably mention both, either as a single line or two lines. We may even want to include details about the new `start_scn`, `start_ts_ms`, and `commit_ts_ms` which represent the SCN when the transaction started, the time the transaction started, and the time the transaction was committed.  wdyt?

Reply:
I defer to your knowledge on this. My change was actually naive -- I was trying to be more technically accurate without realizing that the two were not synonymous. I'm happy to add the additional details, but I'll have to look into the other options more ... unless you can provide a quick summary here?  :)

Reply:
`scn`: the system change number associated with the change event
`start_scn`: the system change number for the change event's transaction start
`start_ts_ms`: the timestamp when the change event's transaction started
`commit_scn`: the system change number for the change event's transaction commit
`commit_ts_ms`: the timestamp when the change event's transaction was committed